<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputeLSS</class>
 <widget class="QDialog" name="ComputeLSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Select data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dataBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Gradient mode type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gradientModeBox">
       <item>
        <property name="text">
         <string>Linear time gradient</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Logarithmic time gradient</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>LSS model name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>222</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="closeButton">
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>okButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>closeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
